E-Commerce/E-Commerce/Presentation/Views/Home/ChartView/CartViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  ChartViewController.swift
+//  E-Commerce
+//
+//  Created by Orlando Nicolas Marchioli on 24/01/2024.
+//
+
+import UIKit
+
+class CartViewController: UIViewController {
+    
+    
+    @IBOutlet weak var cartEmptyImage: UIImageView!
+    @IBOutlet weak var cartTableView: UITableView!
+    var cartViewController: CartViewController?
+    @IBOutlet weak var itemTotal: UILabel!
+    var productsToBuy: [ProductToBuy] = []
+    @IBOutlet weak var confirmButton: UIButton!
+    let verticalCellSpacing: CGFloat = 70.0
+    
```

Comment:
Organize this lines on groups (IBOulets and variables) and mark down the code to make it more easy to navigate. You can find some examples over here: https://www.avanderlee.com/xcode/xcode-mark-line-comment/